src/quill-cursors/cursor.ts

```diff
@@ -77,9 +89,21 @@ export default class Cursor {
     this._el.parentNode.removeChild(this._el);
   }
 
+  public toggleNearCursor(pointX: number, pointY: number): boolean {
+    const {x, y, width, height} = this.coordinates;
+
+    const isXNear = pointX - x - width <= 0 && pointX - x >= 0;
+    const isYNear = pointY - y - height <= 0 && pointY - y >= 0;
+    const shouldShow = isXNear && isYNear;
+
+    this.toggleFlag(shouldShow);
```

Comment:
`toggleFlag()` ([deliberately](https://github.com/reedsy/quill-cursors/pull/50)) ignores the `hideDelay` [option](https://github.com/reedsy/quill-cursors#configuration), but we should honour the `hideDelay` when moving our mouse away.